pkg/crc/pullsecret/pullsecret.go

```diff
@@ -87,7 +88,7 @@ func addpullSecretToInstanceDisk(driver drivers.Driver, pullSec string) error {
 }
 
 func addKubeconfigFileToInstance(driver drivers.Driver, kubeconfigFilePath string) error {
-	kubeconfig, err := ioutil.ReadFile(kubeconfigFilePath)
+	kubeconfig, err := ioutil.ReadFile(filepath.Clean(kubeconfigFilePath))
```

Comment:
Arguably a false positive too

Reply:
Yes, whatever doesn't come as part of user input side mostly those check might be false positive and either we just don't put `filepath.Clean` and put a `nosec` comment. I preferred to have this.